test/integration/scheduler/framework_test.go

```diff
@@ -1885,10 +1884,11 @@ func TestPreemptWithPermitPlugin(t *testing.T) {
 	// }
 
 	if err := wait.Poll(200*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
-		_, err := getPod(testCtx.ClientSet, waitingPod.Name, waitingPod.Namespace)
-		return apierrors.IsNotFound(err), nil
+		w := false
+		permitPlugin.fh.IterateOverWaitingPods(func(wp framework.WaitingPod) { w = true })
+		return !w, nil
```

Comment:
(Probably a bit picky) Can we change the timeout of `TestPermitPlugin` from 10 seconds to 30 seconds; otherwise if the preemption logic lasts more than 10 seconds, we cannot tell if the waitingPod is rejected by preemption logic or the timer.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler/framework_test.go#L453
https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler/framework_test.go#L1852

the timeout is already 30s

Reply:
I see. I checked the UT...